tests/stubtest_allowlists/py3_common.txt

```diff
@@ -272,3 +272,437 @@ os.O_[A-Z_]+
 socket.AF_DECnet
 socket.[A-Z0-9_]+
 (termios.[A-Z0-9_]+)?
+
+# Exists at runtime, but missing from stubs
+__main__.\w+
+_json.encode_basestring
+_markupbase.ParserBase.parse_comment
+_markupbase.ParserBase.parse_declaration
+_markupbase.ParserBase.parse_marked_section
+_markupbase.ParserBase.updatepos
+_thread.LockType.acquire_lock
+_thread.LockType.locked_lock
+_thread.LockType.release_lock
+_thread.RLock
+_thread.allocate
+_thread.exit_thread
+_thread.start_new
+_threading_local._localimpl.localargs
+_threading_local._localimpl.locallock
+asynchat.async_chat.encoding
+asynchat.async_chat.use_encoding
+asynchat.find_prefix_at_end
+asyncore.dispatcher.addr
+asyncore.dispatcher.handle_accepted
```

Comment:
These modules are deprecated in their entirety, and have been removed entirely in the current main branch of Python. That might be reverted before Python 3.11 is released, but we might want to note that here, or move them to a "Won't fix" section